notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>"A geodesic is the shortest path between two points on a manifold."</p><p>--&gt; we cannot say this yet, because we have not defined how we compute the length of a path, thus not how we define the "shortest" path. we cannot define it yet, because we need the notion of Riemannian metric to define notions of lengths. I would thus remove this sentence.</p><p> </p><p> </p><p>Typo: each point and tangent vector</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>I have added a subsection that is called: "intuitive definition of a geodesic" where i say that we cannot yet truly define distnace, but that the intuative definition is the shortest distance between two points.</p>